print/locales/vueI18nConfig.js

```diff
@@ -25,7 +25,7 @@ import rmCHScoutCommon from '@/common/locales/rm-CH-scout.json'
 export default defineI18nConfig(() => ({
   fallbackLocale: {
     rm: ['de'],
-    default: 'en',
+    default: ['en'],
```

Comment:
In frontend (and consequently pdf, which gets it passed in from frontend), it's still just a string: https://github.com/ecamp/ecamp3/blob/devel/frontend%2Fsrc%2Fplugins%2Fi18n%2Findex.js#L41
Why is it no problem there?

Reply:
Because this comes from nuxti18n, which we only use in print
https://github.com/nuxt-modules/i18n/blob/166f5aec3411b8daa448e674ef8a0269f967908a/src/runtime/messages.ts#L45